frontend/src/views/web/challenge/overview.html

```diff
@@ -1,4 +1,14 @@
 <section class="ev-sm-container ev-view challenge-container">
+    <div class="ev-md-container ev-card-panel ev-z-depth-5 ev-challenge-approval-view"
+         ng-if="!challenge.approved_by_admin && challenge.isChallengeHost">
+        <div class="row margin-bottom-cancel">
+            <div class="col s12 m11 l11">Thank you for successfully creating a challenge on EvalAI.
```

Comment:
Lets add some more classes here `fs-16 w-300` to make font size and weight similar to other text